### PR TITLE
chg: fix: add new policies pages path

### PR DESCRIPTION
**Pull request by raphaelmelo**
## Description
Fix: I added the updated urls in pt/termos-de-uso/ - news paths: /privacy-policy and /cookie-policy 

image

### CODE REVIEW

**Review by carlostrr — APPROVED** [review]

File: src/pages/termos-de-uso.js [hunk 1]
```diff
@@ -65,10 +65,10 @@ export default function Contact() {
           </p>
           <ul className="list-disc ml-8">
             <li>
-              <a href="/politica-de-privacidade">Política de Privacidade</a>
+              <a href="/privacy-policy">Política de Privacidade</a>
```
Comment (monuelo) [inline src/pages/termos-de-uso.js:68]: This ref shouldn't point to the English version of this page.
```suggestion
              <a href="/pt/privacy-policy">Política de Privacidade</a>
```

File: src/pages/termos-de-uso.js [hunk 2]
```diff
@@ -65,10 +65,10 @@ export default function Contact() {
...
             </li>
             <li>
-              <a href="/politica-de-cookies">Política de Cookies</a>
+              <a href="/cookie-policy">Política de Cookies</a>
```
Comment (monuelo) [inline src/pages/termos-de-uso.js:71]: This ref shouldn't point to the English version of this page.
```suggestion
              <a href="/pt/cookie-policy">Política de Cookies</a>
```

**Review by monuelo — CHANGES_REQUESTED** [review]
It would help if you didn't point to these paths, leading to a page with a different language.

Consider using Docusaurus [`<Link>`](https://docusaurus.io/docs/docusaurus-core#link) Component instead, or apply the suggestions I've made.

**Comment by raphaelmelo** [comment]
> Done!

**Review by monuelo — APPROVED** [review]

**Pull request merged**